src/components/Reports/ReportEdit.tsx

```diff
@@ -157,8 +170,26 @@ export default function ReportEdit(): JSX.Element {
     }
   };
 
-  const redirectToReportView = () => {
-    history.push(APP_PATHS.REPORTS_VIEW.replace(':reportId', reportId));
```

Comment:
this deletion intended?

Reply:
Yeah, I just moved this out of the `/** Report update functions */` section